src/components/space/ActionsMap.js

```diff
@@ -111,6 +122,13 @@ function ActionsMap() {
                         20,
                       )}px`,
                     }}
+                    onClick={() => handleClusterZoom(longitude, latitude)}
+                    onKeyPress={(event) => {
+                      if (event.keyCode === 13)
```

Comment:
Either factor out `13` to a constant `ENTER_KEY_CODE` or add a comment.

Reply:
👍 👍